src/platform/packages/shared/kbn-unified-metrics-grid/src/components/metrics_experience_grid.tsx

```diff
@@ -152,6 +153,7 @@ export const MetricsExperienceGrid = ({
           ),
         },
       }}
+      toolbarWrapAt={isFullscreen ? 'l' : 'xl'}
```

Comment:
More of a nit, but I would use `m/l` instead, `l/xl` looks a bit too early IMO - I think we still have enough space to fit them together, even with values, or not? wdyt? 

https://github.com/user-attachments/assets/d4fc7c49-4994-4254-8085-aee84066d451

Reply:
The problem with that is when the dimension field name is too long, values can also be too long. You could try the same with `span.destination.service.resource`